src/encoded/static/inline.js

```diff
@@ -1,5 +1,9 @@
 'use strict';
 
+if (window && !window.Promise) {
```

Comment:
It is better to do something like
window.Promise = window.Promise || require('bluebird');
However, I am in risk-avert mode and do not want to risk changing anything unless neccesary.